exonum-java-binding/core/rust/Cargo.lock

```diff
@@ -270,7 +270,7 @@ name = "bzip2-sys"
 version = "0.1.7"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "cc 1.0.38 (registry+https://github.com/rust-lang/crates.io-index)",
+ "cc 1.0.26 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Are cc and backtrace downgraded on purpose?

Reply:
No, it's probably an accidental result of merging. Thanks, going to correct this.